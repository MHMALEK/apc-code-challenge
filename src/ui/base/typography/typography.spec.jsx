import BaseTypography from '.';
import { shallowWithTheme } from '../../../utils/test/test-helper';

describe('BaseIcon Componenent render', () => {
  const sampleName = 'h2';

  const BaseComponent = shallowWithTheme(<BaseTypography variant={sampleName} />);

  it('it should match the SnappShot', () => {
    expect(BaseComponent).toMatchSnapshot();
  });
  it('it should renders the base html element', () => {
    expect(BaseComponent.html()).toContain('h2');
  });
});
