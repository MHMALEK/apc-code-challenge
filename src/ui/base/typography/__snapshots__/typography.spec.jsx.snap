// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseIcon Componenent render it should match the SnappShot 1`] = `<styled.h2 />`;
